### PR TITLE
Feature/maps 293 tabs compoonent

### PR DESCRIPTION
**Pull request by martinmcneela**
BEFORE CREATING THIS PR, have you bumped the package JSON where appropriate?

Ticket / Why this is needed: https://edozohq.atlassian.net/browse/MAPS-293

Link to Figma doc: https://www.figma.com/file/P0HWZYxYUkYMmC5dYeppt0/CAD-data?node-id=259%3A12347

### CODE REVIEW

File: src/Tabs/Panel.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { useTabsContext } from './TabsContext';
+import { StyledPanel } from './Panel.styles';
+
+export interface PanelProps {
+  whenActive: string;
+}
+
+export const Panel: React.FC<PanelProps> = ({ whenActive, children }): any => {
```
Comment (zac-robinson) [inline src/Tabs/Panel.tsx:9]: Why does this return any?

File: src/Tabs/Panel.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { useTabsContext } from './TabsContext';
+import { StyledPanel } from './Panel.styles';
+
+export interface PanelProps {
+  whenActive: string;
```
Comment (zac-robinson) [inline src/Tabs/Panel.tsx:6]: Can this be isActive or something?

File: src/Tabs/Panel.styles.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+interface PanelStyleProps {
+  isActive: boolean;
+}
```
Comment (zac-robinson) [inline src/Tabs/Panel.styles.tsx:5]: These props don't seem to be used?

File: src/Tabs/Tabs.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      </Tabs.TabWrapper>
+      <Tabs.Panel whenActive="tab-1">a panel</Tabs.Panel>
+      <Tabs.Panel whenActive="tab-2">b panel</Tabs.Panel>
+      <Tabs.Panel whenActive="tab-3">c panel</Tabs.Panel>
+    </Tabs>
+  </Story>
+</Preview>
+
+<Preview>
+  <Story name="Three tabs strached">
```
Comment (zac-robinson) [inline src/Tabs/Tabs.stories.mdx:32]: spelling

**Comment by zac-robinson** [comment]
When hovering a tab, i feel the cursor shouldn't be the text cursor

File: src/Tabs/TabsContext.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import { createContext, useContext } from 'react';
+
+interface State {
+  setTab: any;
```
[the remark this answers is not in the document]
Comment (martinmcneela) [reply src/Tabs/TabsContext.ts:4]: It was being a pain in the balls, needs like a dispatch type so I want to loop back to it

File: src/Tabs/Panel.tsx (hunk 2 above)
Comment (martinmcneela) [reply src/Tabs/Panel.tsx:6]: whenActive makes more sense in this case as it is based on the tab that is active. That's why I have changed it up for the style prop.

File: src/Tabs/Tabs.stories.mdx (hunk 4 above)
Comment (martinmcneela) [reply src/Tabs/Tabs.stories.mdx:32]: When you see a spelling mistake can you let me know which word it is? I know it should be obvious, but really not easy for me.

Comment (zac-robinson) [reply src/Tabs/Tabs.stories.mdx]: Strached should be stretched

Comment (martinmcneela) [reply src/Tabs/Tabs.stories.mdx]: aw yeah, I got there eventually, just took me while.

File: src/Tabs/Panel.tsx (hunk 2 above)
Comment (zac-robinson) [reply src/Tabs/Panel.tsx:6]: Can you rename whenActive to be tabId

File: src/Tabs/Tab.tsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useTabsContext } from './TabsContext';
+import { StyledTab, StyledTabWrapper } from './Tab.styles';
+
+export interface TabProps {
+  tab: string;
```
Comment (zac-robinson) [inline src/Tabs/Tab.tsx:6]: Rename tab to be tabId

**Pull request merged**